src/pages/cloud-object-storage/index.tsx

```diff
@@ -83,7 +92,7 @@ export function getServerSideProps(ctx: GetServerSidePropsContext) {
 
```

Comment:
Use locale for all JSON as we have the translations for the page.

Reply:
Done